server.js

```diff
@@ -42,9 +42,56 @@ server.use('/dev', proxy(DEVSERVER))
 server.listen(PORT, () => console.log(`listening on port ${PORT}`))
 
 
+const read_active_filter = ({on_filter}) => {
+    // Step 1: See if there is an active context
+    const tw_context = readline.createInterface({
+        input: spawn_tw(['_get', 'rc.context']).stdout
+    })
+    let context = ''
+    tw_context.on('line', (line) => {
+        if (!context && line) context = line
+    })
+    tw_context.on('close', (line) => {
+        // Step 2: If there is an active context, get its filter.
+        //         If not, get the filter for the "next" report.
+        let cmd = ['_get']
+        if (context)
+            cmd.push('rc.context.' + context)
+        else
+            cmd.push('rc.report.next.filter')
```

Comment:
Hm, shouldn't both apply? What is the logic for applying only the context filter if defined? When you call `task` on the commandline with a context set, it applies both the context filter and the report filter (`next`, `list`, whatever...).

Reply:
A flawed one obviously :)